### PR TITLE
Hide menu

### PR DESCRIPTION
**Pull request by mojcacolnik**

### CODE REVIEW

**Comment by mojcacolnik** [comment]
Before I had the logout button at the bottom of the profile view, then when I implemented conditional rendering of buttons if user is logged in, I moved the logout button to the navbar. At the moment it is the only one not responding to the navbar collapsing at click, because it still has router.push action. Will keep working on it

File: frontend/src/App.vue [hunk 1]
```diff
@@ -1,51 +1,64 @@
 <script>
-import { mapActions } from 'vuex'
-import axios from 'axios'
+import { mapActions, mapState } from "vuex";
+import axios from "axios";
```
Comment (jkuri) [inline frontend/src/App.vue:3]: why did you change single quotes to double quotes? please be consistent and use single quotes everywhere.

File: frontend/src/App.vue [hunk 2]
```diff
@@ -1,51 +1,64 @@
...
-      await this.logout()
-      this.$router.push('/login')
-    }
-  }
-}
-
+      await this.logout();
+      this.$router.push("/login");
+    },
+    hideNavBar() {
```
Comment (jkuri) [inline frontend/src/App.vue:13]: I like this solution 👍

**Review by jkuri — CHANGES_REQUESTED** [review]
This solution works, but now I guess user does not have a option to log out since you removed it from Profile view, please fix that and then this PR is ready to be merged.

**Pull request merged**